packages/auth/src/core/credentials/email.test.ts

```diff
@@ -84,6 +87,57 @@ describe('core/credentials/email', () => {
           password: 'some-password'
         });
       });
+
+      context('#recaptcha', () => {
```

Comment:
Please add some tests for cases where `verifier.verify` rejects (i.e. one test where it rejects first time without the second parameter then succeeds, and one test where it rejects in both cases)